WooCommerce/UITestsFoundation/Screens/Products/ProductsScreen.swift

```diff
@@ -77,14 +93,14 @@ public final class ProductsScreen: ScreenObject {
         return self
     }
 
-    public func tapAddProduct() throws -> Self {
-        app.buttons["product-add-button"].tap()
-        return self
-    }
-
     public func selectProductType(productType: String) throws -> SingleProductScreen {
         let productTypeLabel = NSPredicate(format: "label CONTAINS[c] %@", productType)
         app.staticTexts.containing(productTypeLabel).firstMatch.tap()
         return try SingleProductScreen()
     }
+
+    public func selectSearchButton() throws -> ProductSearchScreen {
```

Comment:
Just a thought: we really tap the button, not select it. We select the product type, though. There are some old functions named `selectAddProduct` and `selectProduct`, which I also think should start with `tap`, but it's not in the PR scope.